ntc_rosetta/parsers/openconfig/junos/__init__.py

```diff
@@ -18,9 +19,17 @@ class JunosParser(parser.RootParser):
 
     class Yangify(parser.ParserData):
         def init(self) -> None:
-            self.root_native["dev_conf"] = etree.fromstring(
-                self.root_native["dev_conf"]
-            )
+            parsed_xml = etree.fromstring(self.root_native["dev_conf"])
+            if parsed_xml.tag != "configuration":
+                parsed_xml = parsed_xml.find("configuration")
+                if parsed_xml is None:
+                    raise AttributeError(
+                        "Unable to locate 'configuration' tag in XML blob"
```

Comment:
We made changes to the testing so if you could rebase off the `develop` branch and change the testing for these change to match the new way, that would be great.

I think we should also start creating our own exceptions for parsing errors rather than use built-in errors. Such as `ntc_rosetta.exceptions.ConfigurationParsingError`, but this needs more discussion.